### ISSUE TITLE
Installing on High Sierra/ Bug in tagger.py, line 48

### ISSUE THREAD

**Issue by GarySwift** [Kraymer/replica#2]
Thanks for making this. I use it all the time. When I am making a playlist, I will occasionally come across a song that is not  320kbps. I then replace the entire album using **Replica**.

I know that this is a bit old so I am not giving you hassle for it. However, I had a lot of trouble installing on the latest masOS, High Sierra 10.13.4.

The first issue was with trying to get this working with the version of Python that comes with macOS. When I installed pip, I got a lot of warnings including.

```
gary@~$ sudo easy_install pip
...
matplotlib 1.3.1 requires nose, which is not installed. 
matplotlib 1.3.1 requires tornado, which is not installed.
...
```

These errors were preventing me form installing Replica. Using pip, I installed nose and tornado. This allowed me to install Replica but I kept getting the following:

```
-bash: replica: command not found
```

There was a lot of trouble shooting on my part. I won’t overload this with terminal outputs because nothing was working for me. 

After much frustration, I followed some internet advice and installed **Python 3**. I then installed **Replica** using **pip3**.

```
pip3 install replica
```

I then checked it was installed using `freeze`.

```
gary@~$ pip3 freeze
mutagen==1.40.0
replica==0.1.1
gary@~$

```

I was still getting the “command not found” error when trying to run Replica so I restarted terminal. I then ran replica again and it tried to run but there was an error using replica and python 3. Python 3 is strict about using parenthesis with the print statement.

Line 48, [tagger.py](https://github.com/Kraymer/replica/blob/master/replica/tagger.py) needs to be:

```
print (msg)
```  

## TLDR
Install latest version of [Python](https://www.python.org/downloads/).

Use pip3 to stall replica. 

```
pip3 install replica
```

Restart terminal.

Fix error on line 48, [tagger.py](https://github.com/Kraymer/replica/blob/master/replica/tagger.py).

```
print msg
```  

needs to be:

```
print (msg)
```

**Comment by Kraymer** [comment]
Thanks, should be fixed in 0.1.2 version.

**Comment by Kraymer** [comment]
> I know that this is a bit old so I am not giving you hassle for it.

Héhé right, I don't use it personally anymore. But if you have a feature request, don't hesitate !
